src/style.css

```diff
@@ -0,0 +1,31 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  flex-direction: row;
+}
+
+.stars--1 :first-child,
+.stars--2 :nth-child(-n+2),
+.stars--3 :nth-child(-n+3),
+.stars--4 :nth-child(-n+4),
+.stars--5 :nth-child(-n+5) {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars :nth-child(n+1):not(:last-child) {
+  margin: 0 4px 0 0;
+}
+
+.stars__star {
+  background-image: url("./images/star.svg");
+  background-repeat: no-repeat;
+  list-style: none;
+  width: 16px;
+  height: 16px;
+  background-position: center;
```

Comment:
Pay attention to your css properties order
https://9elements.com/css-rule-order/

Reply:
cool! thank i realy didn't know that

Reply:
if I understood correctly...